### PR TITLE
Release 7.0.2

### PR DESCRIPTION
**Pull request by JordanBalesBAO**

### CODE REVIEW

**Review by flopez-bao — APPROVED** [review]
Tested locally in the app with this branch. Ran cop24 testing malawi datapack and unpacked without stop errors. Ran a couple of cop23 datapacks as well.

File: R/datapackVsDatim.R
Comment (jacksonsj) [inline R/datapackVsDatim.R:1]: I largely don't understand the context for the changes in this file, so we'll need someone else to review.

File: tests/testthat/test-createTestDataSet.R
Comment (jacksonsj) [inline tests/testthat/test-createTestDataSet.R:1]: I'm also not super familiar with this code, so would need someone else to review.

**Review by jacksonsj — APPROVED** [review]
Other than the two files I don't know much about that I noted in my comments, approved.

File: tests/testthat/test-create-schema.R [hunk 1]
```diff
@@ -174,7 +174,8 @@ with_mock_api({
     expect_type(test_dataset$FY, "double")
     expect_type(test_dataset$period, "character")
     #load(getRDA("cop22_data_pack_schema"))
-    expect_identical(test_dataset, cop22_data_pack_schema)
+    #expect_identical(test_dataset, cop22_data_pack_schema)
+    # Seems to no longer work with list columns
```
Comment (jason-p-pickering) [inline tests/testthat/test-create-schema.R:178]: Why was this commented out?

**Review by jason-p-pickering — COMMENTED** [review]
Hold off until we can determined why these changes were made in the unit tests.

File: tests/testthat/test-create-schema.R (hunk 1 above)
Comment (JordanBalesBAO) [reply tests/testthat/test-create-schema.R:178]: I believe this happened at commit 6c1959e3f2454c2fdd39a5efc07d3ef2871cc9f7. Looks to be related to https://jira.pepfar.net/browse/DP-1123. Outside of that I think we will have to ask @jacksonsj

Comment (JordanBalesBAO) [reply tests/testthat/test-create-schema.R:178]: @jason-p-pickering @jacksonsj what is the plausibility of us sorting this out before EOW?

**Pull request merged**